src/client/components/Fields.tsx

```diff
@@ -259,12 +258,12 @@ export function createTextArea(name: string, placeholder: string,
   );
 }
 
-export function createMonthPicker(name: string = null) {
+export function createMonthPicker(name = 'birthdateMonth') {
```

Comment:
nit: Removed type definition?

Reply:
Typescript automatically infers the type from the default parameter. Type definition wasn't removed, just changed.

![image](https://user-images.githubusercontent.com/1966472/74779394-406e7c80-5252-11ea-9557-c93766ab564c.png)